src/ec/packages/ec-component-inpage-navigation/inpage-navigation.story.js

```diff
@@ -0,0 +1,49 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+import iconPath from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+
+import inpageNavigationDocs from './docs/inpage-navigation.md';
+
+import demoData from './demo/data';
+import pageFiller from './demo/page-filler';
+
+import inpageNavigation from './inpage-navigation.html.twig';
+
+storiesOf('Components/Inpage navigation', module)
```

Comment:
```suggestion
storiesOf('Components/Navigation/Inpage navigation', module)
```

the component should be listed under "Navigation" ;)